ddl/ddl_worker.go

```diff
@@ -185,28 +185,62 @@ func buildJobDependence(t *meta.Meta, curJob *model.Job) error {
 	return nil
 }
 
-// addDDLJob gets a global job ID and puts the DDL job in the DDL queue.
-func (d *ddl) addDDLJob(ctx sessionctx.Context, job *model.Job) error {
+func (d *ddl) limitDDLJobs() {
+	for {
+		tasks := make([]*limitJobTask, 0, batchAddingJobs)
```

Comment:
This line should be moved out of the loop.

Reply:
Done.